services/navigation/builtin/builtin.go

```diff
@@ -18,13 +18,25 @@ import (
 	"go.viam.com/rdk/resource"
 	"go.viam.com/rdk/services/motion"
 	"go.viam.com/rdk/services/navigation"
+	"go.viam.com/rdk/services/vision"
 	"go.viam.com/rdk/spatialmath"
 	rdkutils "go.viam.com/rdk/utils"
 )
 
 const (
 	metersPerSecDefault = 0.5
 	degPerSecDefault    = 45
+
+	// how far off the path must the robot be to trigger replanning.
+	planDeviationMetersDefault = 1.
```

Comment:
Our convention has been naming things like `defaultPlanDeviationM`, so this should be done for all the defaults here (as well as the two that are above that you did not change for this PR)  

Additionally, this one already has a default that we set to math.Inf(1) for now, so we should reflect that here

Reply:
ok, however I cannot do 
```
defaultPlanDeviationM = math.Inf(1)
```
since
```
math.Inf(1) (value of type float64) is not constant
```